CHANGELOG.md

```diff
@@ -7,8 +7,7 @@ The format is based on [Keep a Changelog 1.0.0].
 ## [Unreleased]
 - Added `SearchParameters` dataclass for use with search functions using the legacy kwargs from `Client.advanced_search` and new `collections` field for filtering by collections
 - Changed `Client.advanced_search` interface to take in `SearchParameters` as opposed to the legacy kwargs.
-- Added `search_and_parse_results` and `search_judgments_and_parse_results` methods to `Client`
-- Added `SearchResults` class to encapsulate document search results
```

Comment:
This gets added back in https://github.com/nationalarchives/ds-caselaw-custom-api-client/pull/223/files